weekThree/tests/test_file_manager.py

```diff
@@ -0,0 +1,14 @@
+import pytest
+from weekThree.file_manager import FileManager
+
+def test_count_lines(tmp_path):
```

Comment:
While tests cover basic functionality, expand testing to include more edge cases and ensure file operations in tests are isolated (using temporary directories or pytest fixtures) to prevent side effects (by generating a temporary folders/files by Pytest).